avocado/utils/software_manager.py

```diff
@@ -409,6 +411,7 @@ def __init__(self, cmd='yum'):
         cmd_result = process.run(y_cmd, ignore_status=True,
                                  verbose=False, shell=True)
         out = cmd_result.stdout.strip()
+        out = astring.to_text(out)
```

Comment:
why not we just use ```out = cmd_result.stdout_text.strip() ```

Reply:
stdout_text from avocado-vt project, we need avoid circular import

Reply:
This fix will not back-port into avoado-52lts, but for master only. So I think it will not break down any user. If I'm wrong, pls correct me freely.

Reply:
Well the `astring.to_text` is also not available in lts, so using it doesn't fix the problem. It looks like we have to keep using bytes in utils until we get new lts.

Reply:
... what I mean is it'd be safer to use `re.findall(b'\\d*.\\d*.\\d*', ...)` and adopting the `log.debug`, or decoding it manually using `ver.decode()`. Using ascii should be fine there as we only detect int.int.int.

Reply:
Yes, actually I'd recommend using the `bytes` in regexp and follow it by `.decode()` which should turn the result safely into string.

Reply:
I assume that 52lts code is locked down, and the change is integrated into master, and doesn't impact 52lts.

Reply:
52lts is not supposed to be used with python3 so it shouldn't really matter. Yep, we need to convert the output in case of exception, so perhaps it's not a bad idea to do it first. Based on the usage I'd probably use `decode(encoding, errors='replace')` to avoid crashes in case of bad encoding.